test/e2e/bootstrapper_test.go

```diff
@@ -161,3 +176,24 @@ func isKubeletRunning(t *testing.T, logPath string) bool {
 	assert.NoError(t, err)
 	return strings.Contains(string(buf), "Started kubelet")
 }
+
+// testPathInKubeletArgs checks if the paths given as arguments to kubelet service are correct
+// Only checks for paths that are dependent on the underlying OS
+func testPathInKubeletArgs(t *testing.T, checkPathsFor []string) {
+	// Get fully qualified path for kubelet
+	_, path, err := getSvcInfo(bootstrapper.KubeletServiceName)
+	require.NoError(t, err, "Could not get kubelet arguments")
+	// Split the arguments from kubelet path
+	kubeletArg := strings.Split(path, " ")
+	for _, arg := range kubeletArg {
+		split := strings.Split(arg, "=")
+		// Ignore single valued arguments
```

Comment:
What about arguments that have multiple `=`?

Reply:
Good Point. Replacing with SplitN.